testnet-contracts/contracts/CosmosBridge.sol

```diff
@@ -149,22 +149,37 @@ contract CosmosBridge {
         // Increment the prophecy claim count
         prophecyClaimCount = prophecyClaimCount.add(1);
 
-        address originalValidator = msg.sender;
-
         ClaimType claimType;
+        address tokenAddress;
         if (_claimType == ClaimType.Burn) {
+            require(
+                bridgeBank.hasLockedFunds(_tokenAddress, _amount),
+                "Not enough locked Ethereum assets to complete the proposed prophecy"
+            );
             claimType = ClaimType.Burn;
+            tokenAddress = _tokenAddress;
```

Comment:
_tokenAddress is no longer available in burn/lock relays, so I think we need to use bridgeBank.getControlledBridgeToken for the burn too. can discuss on slack if needed.

Reply:
Removed `_tokenAddress` in latest.